backend/src/main/java/com/huellapositiva/application/dto/ChangePasswordDto.java

```diff
@@ -0,0 +1,28 @@
+package com.huellapositiva.application.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class ChangePasswordDto {
+
+    @NotEmpty(message = "You must to write something")
+    @NotBlank(message = "The password is not blank space")
+    @Size(min = 6, message = "The password is too short")
+    @Pattern(regexp = "^[a-zA-Z0-9àèìòùÀÈÌÒÙáéíóúýÁÉÍÓÚÝâêîôûÂÊÎÔÛãñõÃÑÕäëïöüÿÄËÏÖÜŸ.,:;+*/|%!@#$&€=<>() -]*$", message = "The new password does not match with the regular expresion")
```

Comment:
No need for tilde characters. At the time of creating one account you are normally only asked to provide common letters, numbers and common characters.

Consider https://docs.oracle.com/cd/E11223_01/doc.910/e11197/app_special_char.htm#MCMAD416

Reply:
We have changed the regex according to oracle.